disasm-test/Main.hs

```diff
@@ -512,6 +652,23 @@ processBitCode pfx file = do
         when det $ liftIO $ putStrLn $ "## parsed Bitcode to " <> parsed
         return (parsed, Nothing)
 
+
+postParseTests :: AST.Module -> TestM ()
+postParseTests m = ensureValidMetadataIndices m
+  where
+    ensureValidMetadataIndices md = do
+      let idxs = AST.umIndex <$> AST.modUnnamedMd md
+      let uniqIdxs = nub idxs
+      let numDups = length idxs - length uniqIdxs
+      unless (numDups == 0)
+        $ do Details det <- gets showDetails
+             when det $ liftIO
+               $ putStrLn $ "Unnamed MetaData indices: " <> show idxs
```

Comment:
I'm not sure if I've ever seen "metadata" capitalized in CamelCase before. I'd propose just using:

```suggestion
               $ putStrLn $ "Unnamed metadata indices: " <> show idxs
```

Reply:
I was trying to match the internal variable name, but missed.  I've clarified these references.